src/libra_py/workflows/nbra/compute_properties.py

```diff
@@ -50,7 +49,7 @@ def compute_properties_dia_gamma(params, es_curr, es_next, curr_index):
 
     os.system("mkdir %s" % rd)
     #========== Print out ================
-    S.real().show_matrix("%s/S_dia_ks_%d_re" % (rd, curr_index))
+    #S.real().show_matrix("%s/S_dia_ks_%d_re" % (rd, curr_index))
```

Comment:
perhaps, we could enable the user to customize these prefixes.. Not urgent though.